src/AMSlib/include/AMS.h

```diff
@@ -8,6 +8,8 @@
 #ifndef __AMS__
 #define __AMS__
 
+#include <cstdint>
```

Comment:
is this used?

Reply:
Added back int64_t. So now it needs to be there.